docs/_sidebar.md

```diff
@@ -13,9 +13,9 @@
   - [Proofreading a Curriculum File](how-to-proofread-files.md)
   - [How to Translate the Website](how-to-translate-the-website.md)
 - **Optional Guides**
-  - [Catch outgoing emails locally](how-to-catch-outgoing-emails-locally.md)
-  - [Set up freeCodeCamp on WSL](how-to-setup-wsl.md)
+  - [Set up freeCodeCamp on Windows(WSL)](how-to-setup-wsl.md)
```

Comment:
```suggestion
  - [Set up freeCodeCamp on Windows Subsystems for Linux](how-to-setup-wsl.md)
```

Just a thought, but I wonder if we should spell this out for translation purposes?

Reply:
I am concerned if this fits on the sidebar. It's already spelt out in the guide in the title and elsewhere.

<details>
<summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/1884376/110678047-de393e80-81fb-11eb-98e6-6d720deedcf7.png)


</details>

Reply:
Ah, true - a good point.

Reply:
Good to resolve and review then?